src/dynamic_reconfigure/parameter_generator_catkin.py

```diff
@@ -581,11 +581,13 @@ def _rreplace_str_with_val_in_dict(self, orig_dict, old_str, new_val):
             if isinstance(v, dict):
                 self._rreplace_str_with_val_in_dict(v, old_str, new_val)
             elif isinstance(v, list):
+                idx = 0
```

Comment:
good point, forgot that was a thing. Thanks! https://github.com/ros/dynamic_reconfigure/pull/77/commits/03468be6d6036fb98c00a67ecbc149717f4153a8